OasisDEX/oasis-borrow
# [7402] Don't reuse DPM proxy from empty position

# [Don't reuse DPM proxy from empty position](https://app.shortcut.com/oazo-apps/story/7402/reused-id-s-are-counted-as-individual-positions)

<please insert a shortcut link above>
  
## Changes 👷‍♀️

- previously a proxy had a position if the proxy had value locked in Aave.  now it checks for position created event.
- prevents reuse of proxies of closed or emptied positions.
  
## How to test 🧪

- ensure bug in https://app.shortcut.com/oazo-apps/story/7402/reused-id-s-are-counted-as-individual-positions is resolved
- ensure DS proxy continues to work as normal - see https://docs.google.com/document/d/1aQDSnX93DdkHwvE_IuOu6j1kfZ5bQp0Ef6J17xINahI/edit#
